package.json

```diff
@@ -77,15 +77,20 @@
     "react-router-dom": "5.2.0",
     "react-scripts": "3.4.1",
     "redux": "4.0.5",
-    "typescript": "3.9.6",
+    "typescript": "^3.9.6",
```

Comment:
Why do you unpin the typescript-dependency?

Reply:
I blame the IDE, but done